### PR TITLE
[RUBY-148] Integration tests for local_one default consistency

### PR DESCRIPTION
**Pull request by kishkaru**

### CODE REVIEW

File: integration/session_test.rb [hunk 1]
```diff
@@ -648,4 +648,27 @@ def test_raise_error_on_future_resolution_timeout
...
+  # simple query and verifies through the execution_info that the fulfilled consistency was local_one.
+  #
+  # @since 3.0.0
+  # @jira_ticket RUBY-148
+  # @expected_result local_one should be used as the default consistency
+  #
+  # @test_category consistency
+  #
+  def test_default_consistency_local_one
+    begin
```
Comment (stamhankar999) [inline integration/session_test.rb]: Same begin/end thing.

**Comment by stamhankar999** [comment]
Looks good! Fix the begin/end thing, and merge!

**Comment by avalanche123** [comment]
please rebase this onto 3.0.0b

**Comment by kishkaru** [comment]
Fixed above issue and rebased.

**Comment by stamhankar999** [comment]
:+1: Revised PR looks good to me.

**Pull request merged**